src/main/java/com/mercadolibre/bootcampmelifrescos/model/Cart.java

```diff
@@ -0,0 +1,24 @@
+package com.mercadolibre.bootcampmelifrescos.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name="cart")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class Cart {
+    @Id
```

Comment:
```suggestion
    
    @Id
```

Só por padronização mesmo.. deixa um espaço aqui.